src/main/java/carpetextra/mixins/AnimalMateGoalMixin.java

```diff
@@ -1,17 +0,0 @@
-package carpetextra.mixins;
-
-import carpetextra.CarpetExtraSettings;
-import net.minecraft.entity.ai.goal.AnimalMateGoal;
-import net.minecraft.entity.passive.AnimalEntity;
-import net.minecraft.server.network.ServerPlayerEntity;
-import org.spongepowered.asm.mixin.Final;
-import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.Shadow;
-import org.spongepowered.asm.mixin.injection.At;
-import org.spongepowered.asm.mixin.injection.Inject;
-import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
-
-@Mixin(AnimalMateGoal.class)
-public abstract class AnimalMateGoalMixin {
-    //intentionally left blank
-}
```

Comment:
Hm... This says "intentionally left blank", so it may have a reason to exist. IIRC this was, in its time to prevent NPEs in dispenser feeds animals rules, though it was changed to more specific mixins/just other mixins at some point.

Reply:
This mixin wasn't even defined in carpet-extra.mixins.json, plus it has no methods and does nothing, like the WetSpongeBlockMixin that I also removed

Reply:
The reason it was 'intentionally left blank' was to help in switching from 1.15 (then master) and 1.16 (then future) branches. Just forgot to remove it when 1.16 was the only version.

In general, my experience is that if Mojang implements some of the carpet features in a snapshot, its better to 'comment out' the feature then remove all the signs of it in the cod cause then when you switch back and forth between master and future branches that causes merge conflicts.